src/app/loadout/loadout-ui/menu-hooks.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { getHashtagsFromString } from 'app/inventory/note-hashtags';
+import { Loadout } from 'app/loadout/loadout-types';
+
+// Mock the dependencies
+jest.mock('app/inventory/note-hashtags');
+jest.mock('app/manifest/selectors');
+jest.mock('app/loadout-analyzer/hooks');
```

Comment:
I really, really, really hate mocking. Instead, factor out a function for computing loadoutsByHashtag and test just that.